ui/src/app/streams/streams-deploy/streams-deploy.component.html

```diff
@@ -0,0 +1,58 @@
+<div *ngIf="streamDefinitions">
+  <div class="modal-content" *ngIf="!selectStreamDefinition" id="panel-stream">
+    <div class="modal-header">
+      <h4 class="modal-title pull-left">Confirm Deployment of Stream Definitions</h4>
```

Comment:
I guess this is `Stream Definition` instead of `Stream Definitions` since we can deploy one stream from the modal.

Reply:
Apparently, this does support multiple stream definitions. Hence, this should be fine as is :-)